requirements/specifications/lifecycle/index.md

```diff
@@ -1,274 +1,365 @@
 
-There is no notification for this state, it just is.
+Apps in this state **MUST NOT** be visible.
 
-### 2.2. Inactive
+Apps in this state **MUST NOT** receive RCU key presses.
 
-This state allows an app to be running, but not in one of the two active
-states, i.e. foreground and background. Inactive apps can execute code,
-but are not visible, do not receive any input from RCUs, and cannot use
-the video decoder. Apps should free up memory and other resources in
-this state.
+Apps in this state **MUST NOT** have access to the audio-video decoder. 
 
-When an app transitions to this state, the platform **MUST** dispatch
-the Lifecycle.onInactive notification with the current state and
-previous states.
+Apps in this state **MUST NOT** use the media pipeline.
 
-Apps in this state **MUST** not use the media pipeline and the platform
-**MUST** tear down any Media Pipeline sessions associated with this app.
+### 2.2. Running
+
+This state allows an app to be running and ready to go, but not actively part
+of the user-perceptible experience.
+
+Running apps can execute code, but are not [Presented](./presentation.md) to the user, do not receive
+any input from RCUs, and cannot use the video decoder.
+
+Apps in this state **MUST** have access to the graphics compositor.
+
+Apps in this state **MUST NOT** be visible.
 
-When an app is initializing, and registers for Lifecycle.onInactive
-notifications, the platform **MUST** transition the app to the inactive
-state with the previous state set to initializing, since it is now able
-to execute any code related to that transition.
+Apps in this state **MUST NOT** receive RCU key presses.
 
-If an app is still initializing, and calls Lifecycle.ready(), the
-platform **MUST** transition the app to the inactive state with the
-previous state set to initializing.
+Apps in this state **MUST NOT** have access to the audio-video decoder.
 
-When an app is in either *active* state and is *closed* then the
-platform **MUST** transition the app to the inactive state with the
-previous and current state set correctly.
+Apps in this state **MUST NOT** use the media pipeline and the platform
+**MUST** tear down any Media Pipeline sessions associated with this app.
 
-When an app is in the suspended state and is then unsuspended, then the
-platform **MUST** transition the app to the inactive state with the
-previous state set to suspended.
+### 2.3. Active
 
-### 2.3. Foreground
+This state allows an app to be presented as part of the user-perceptible
+experience.
 
-This state allows an app to be presented as the primary experience to
-the user.
+To determine if an app is the *primary* experience, e.g. fullscreen,
+etc., see [Presentation](./presentation.md).
 
-The platform **MUST** ensure that only one or zero apps is ever in this
-state at one time.
+It is a platform decision as to how many apps may be in this state at
+once.
 
-Apps in this state **MUST** be visible.
+Apps in this state **MAY** be visible.
```

Comment:
Depending on Display value in [Presentation](./presentation.md) the App will be visible in this state or not